app/src/main/java/chat/rocket/android/server/infraestructure/MemoryMessagesRepository.kt

```diff
@@ -7,6 +7,14 @@ import kotlinx.coroutines.experimental.withContext
 
 class MemoryMessagesRepository : MessagesRepository {
 
+    private var lastSyncDate: Long? = null
+
+    override suspend fun saveLastSyncDate(it: Long) {
```

Comment:
It's better to use the same argument name here as defined on the interface: `currentTimeMillis `

Reply:
agree